src/cf-layout/src/cf-layout.less

```diff
@@ -346,16 +346,35 @@
 
     &__flush-top {
         margin-top: 0 !important;
+
+        &.block__border,
+        &.block__border-top {
+            border-top: none;
+        }
     }
 
     &__flush-bottom {
         margin-bottom: 0 !important;
+
+        &.block__border,
+        &.block__border-bottom {
+            border-bottom: none;
+        }
     }
 
     &__flush-sides {
         margin-right: -(@grid_gutter-width / 2);
         margin-left: -(@grid_gutter-width / 2);
 
+        .respond-to-max(@bp-xs-max, {
```

Comment:
Sorry, what I meant was do we need the media query at all? I'm failing to understand why we would want the borders back on a flush-sides after it reaches the `@bp-sm-min` breakpoint. Possibly related, should this be updated to `none`? https://github.com/cfpb/capital-framework/pull/260/files#diff-5140a128023e2d3dcfd06c7e5b80e2c5R315